x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -90,17 +89,11 @@ export const agentsWithEndpointsTotalError = (state: Immutable<EndpointState>) =
   state.agentsWithEndpointsTotalError;
 
 export const endpointsTotalError = (state: Immutable<EndpointState>) => state.endpointsTotalError;
-const queryStrategyVersion = (state: Immutable<EndpointState>) => state.queryStrategyVersion;
 
 export const endpointPackageVersion = createSelector(endpointPackageInfo, (info) =>
   isLoadedResourceState(info) ? info.data.version : undefined
 );
 
-export const isTransformEnabled = createSelector(
```

Comment:
_maybe_ we keep this, and in the event it returns `false`, we display an error that transforms are a requirement (somewhere)

Reply:
kept this removed since we 🪓  MetadataQueryStrategyVersions so didn't seem straight forward anymore.